cms/standard_pages/models.py

```diff
@@ -38,3 +40,86 @@ class InformationPage(BasePage):  # type: ignore[django-manager-missing]
         index.SearchField("summary"),
         index.SearchField("content"),
     ]
+
+
+class IndexPage(BasePage):  # type: ignore[django-manager-missing]
+    template = "templates/pages/index_page.html"
+
+    parent_page_types: ClassVar[list[str]] = ["home.HomePage", "IndexPage"]
+    subpage_types: ClassVar[list[str]] = ["IndexPage", "InformationPage"]
+
+    summary = models.TextField()
+    featured_items = StreamField(
+        [("featured_item", RelatedContentBlock())],
+        help_text="Leave blank to automatically populate with child pages.",
+        blank=True,
+    )
+
+    content = RichTextField(features=settings.RICH_TEXT_BASIC, blank=True)
+    related_links = StreamField([("related_link", RelatedContentBlock())], blank=True)
+
+    content_panels: ClassVar[list["Panel"]] = [
+        *BasePage.content_panels,
+        FieldPanel("summary"),
+        FieldPanel("featured_items"),
+        FieldPanel("content"),
+        FieldPanel("related_links"),
+    ]
+
+    search_fields: ClassVar[list[index.SearchField | index.AutocompleteField]] = [
+        *BasePage.search_fields,
+        index.SearchField("summary"),
+        index.SearchField("content"),
+    ]
+
+    def get_formatted_items(self, request: "HttpRequest") -> list[dict[str, str]]:
+        formatted_items = []
+
+        if featured_items := self.featured_items:
+            for featured_item in featured_items:
+                formatted_items.append(
+                    {
+                        "featured": "true",
+                        "title": {"text": featured_item.value.link["text"], "url": featured_item.value.link["url"]},
+                        "description": featured_item.value["description"],
+                    }
+                )
+
+        else:
+            for child_page in self.get_children().live().public().specific().defer_streamfields():
+                formatted_items.append(
+                    {
+                        "featured": "true",
+                        "title": {
+                            "text": child_page.listing_title or child_page.title,
```

Comment:
```suggestion
                            "text": getattr(child_page, "listing_title", "") or child_page.title,
```

on the off chance that it doesn't inherit from `BasePage`